snapshots/homicide/2024-10-30/unodc.xlsx.dvc

```diff
@@ -0,0 +1,45 @@
+
+      The statistical definition contains three elements that characterize the killing of a person as “intentional homicide”:
+
+      1. The killing of a person by another person (objective element).
+
+      2. The intent of the perpetrator to kill or seriously injure the victim (subjective element).
+
+      3. The unlawfulness of the killing (legal element).
+
+      For recording purposes, all killings that meet the criteria listed above are to be considered intentional homicides, irrespective of definitions provided by national legislations or practices.  Killings as a result of terrorist activities are also to be classified as a form of intentional homicide.
+
+      In order to compile consistent time series of total homicides back to 1990, in several cases data from multiple sources were combined to expand the number of available years within a country’s time series. Time series adjustments were performed when a country had two sources coveringdifferent year-ranges, which had very similar trends in an overlapping time period, but where these trends were at different levels.
+
+      The countries for which adjusted series for total homicide counts prior to the year 2000 have been produced were the following: Belgium, Brazil, China, Ecuador, Germany, Netherlands, New Zealand, Portugal, South Korea, Spain, Thailand, and United Kingdom.
+
+    date_published: "2024-05-16"
+
+    # Citation
+    producer: United Nations Office on Drugs and Crime
+    citation_full: |-
+      UNODC (2024), UNODC Research - Data Portal – Intentional Homicide. https://dataunodc.un.org/dp-intentional-homicide-victims (Accessed on [30 10 2024]).
+    attribution_short: UNODC
+
+    # Files
+    url_main: https://dataunodc.un.org/dp-intentional-homicide-victims
+    url_download: https://dataunodc.un.org/sites/dataunodc.un.org/files/data_cts_intentional_homicide.xlsx
+    date_accessed: 2024-10-30
+
+    # License
+    license:
+      name: CC BY 4.0
```

Comment:
I'm leaving this comment because I am always wondering about this: 

The UN ODC itself does not explicitly release their materials under a CC BY 4.0 license (at least not under their terms-of-use link), but the Terms of Use do seem very similar to the conditions of this license. But at the same time the unodc.org and the dataunodc.un.org website have a copyright notice in the footer which links to this blanket UN copyright notice (https://www.un.org/en/about-us/copyright). 

With UN data sets where they don't explicitly release it under a license I always put "custom license" - © United Nations and then link the terms of use, where they allow the distribution. But I don't know what's the best/ correct way to do it, so I'd like your input!

Reply:
I think what you do is very much the correct way to do things. I think with big organizations like WHO and UN, their data is normally just open by default, so they haven't thought too much about the licensing side of things, so it's hard to know what exactly to put. But I think using `© United Nations`, which they at least mention in their copyright, is better than giving a license they don't mention. Thanks for catching it!

Reply:
Thanks for the input! Yeah, their licensing is a bit confusing sometimes, so it's good to know what you think 👍